### ISSUE TITLE
Injected reviewed-by tags don't follow encoding in the Content-Type

### ISSUE THREAD

**Issue by oohal** [getpatchwork/patchwork#295]
The downloaded mbox has the various Reviewed-by tags from the thread injected into it without checking if the injected text follows the same encoding as the source material. [This patch](http://patchwork.ozlabs.org/patch/1158366/) was sent as us-ascii while the reply was utf-8. The resulting mbox you get from patchwork contains this:

```
Content-Type: text/plain; charset="us-ascii"

*stuff*

Signed-off-by: Oliver O'Halloran <snip>
Reviewed-by: Cédric Le Goater <snip>
```

Which causes `git-am` to barf on the file since the `é` can't be decoded as us-ascii. I'm not sure what the real fix is here. Fixing the charset in the content type to utf-8 would work fine in this case, but it'll probably break if more exotic encodings are in use.

**Comment by daxtens** [comment]
I think this should already be fixed by: dde885c6c2f5 ("mbox: do not copy
Content-Type into exported mbox")?

I don't know if jk has applied that to ozlabs patchwork: I did send it to
him.

Regards,
Daniel

**Comment by stephenfin** [comment]
Correct. We just need to wait for that to be updated, unfortunately. Will close until that's happened, hopefully sooner rather than later.
